bin/webpack-dev-server.js

```diff
@@ -331,9 +331,12 @@ function processOptions(wpOpt) {
 
 	if(argv["open"] || argv["open-page"]) {
 		options.open = true;
-		options.openPage = argv["open-page"] || "";
```

Comment:
The change I made to that line only makes sense in the context that it goes hand right along with the next if statement.

Before my change, if you go through this function with argv for open, you get options.open set to true and options.openPage set to either the open-page command line arg or to empty string.  After my change, this logic is identical except the setting openPage to empty string is now a few lines down instead of being inline.  Basically, the line you are concerned about is irrelevant to the change: I removed an unnecessary condition from the code for a tiny bit of code simplification but, if I had left the line in, the behavior would be the same by the time you finish the function.

Before my change, if you go through this function with options.open already set to true and no argv, you would have options.openPage set to undefined because you literally never hit the line of code you are asking about. **This** is what is causing the bug.  After my change, I check for options.open independent of whether you have argv.open or not and set undefined options.openPage to empty string.

Is that clearer?  I am sorry if I am not being clear. Written communication is not my strongpoint.

Run through the code in your head before and after with argv set to `{}` but options.open set to true.  That is the only case I am changing at all and is the case I am fixing.
The reason it matters if openPage is undefined or empty string is because [later in the file](https://github.com/coltonw/webpack-dev-server/blob/ebd9d9db571a5a173a77ab4a52c6071be09371f4/bin/webpack-dev-server.js#L452) we string concatenate the uri with openPage.